src/test/java/io/camunda/zeebe/bpmnassert/util/Utilities.java

```diff
@@ -151,6 +145,12 @@ public static PublishMessageResponse sendMessage(
 
   public static void increaseTime(final InMemoryEngine engine, final Duration duration) {
     engine.increaseTime(duration);
+    try {
+      // we need to wait some physical time so that InMemoryEngine has a chance to fire the timers
+      Thread.sleep(50);
```

Comment:
💭 This could mean that the timer delay is too small. If we wait just a little bit longer this sleep would not be necessary and I would expect `waitForIdleState` would take care of this.

Reply:
We could make it bigger, but then we would penalize all steps. And since this is a rare use case, I wanted to avoid that.

But happy to be overruled if you have a strong preference here.